src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md

```diff
@@ -40,22 +40,19 @@ finalized yet, the Chrome Team is actively looking for enthusiastic developers
 [feedback on the spec](https://github.com/WebBluetoothCG/web-bluetooth/issues) and
 [feedback on the implementation](https://bugs.chromium.org/p/chromium/issues/entry?components=Blink%3EBluetooth).
 
-A subset of the Web Bluetooth API is available in Chrome 56 for Chrome OS,
-Chrome for Android M, and Mac. This means you should be able to
+A subset of the Web Bluetooth API is available in Chrome OS,
+Chrome for Android M, Mac, and Windows 10. This means you should be able to
```

Comment:
Please tell what us what versions of Chrome go support. The word 'now' doesn't age well.